### PR TITLE
Add new mobile logotypes and export both new and current mobile logotypes

### PR DESCRIPTION
**Pull request by matzimowski**
no visual changes to style-guide docs.

we are porting mobile logotypes from https://github.com/brainly/style-guide-next repo.
they are named with a `new`  postfix, but at some point gonna replace current mobile logotypes.

Screenshot 2019-12-03 at 14 57 58
Screenshot 2019-12-03 at 14 58 18
Screenshot 2019-12-03 at 14 58 28
Screenshot 2019-12-03 at 14 58 39

### CODE REVIEW

File: src/components/logo/Logo.jsx [hunk 1]
```diff
@@ -32,9 +48,17 @@ export const TYPE = {
 
 export const LOGOS = {
   [TYPE.BRAINLY]: 'brainly-761d75d6ea',
+  [TYPE.BRAINLY_MOBILE]: 'brainly-mobile-d9f7a24c8b',
+  // [TYPE.BRAINLY_MOBILE_NEW]: 'brainly-mobile-new-6879551770',
```
Comment (matzimowski) [inline src/components/logo/Logo.jsx]: for new icons fingerprints are missing, we are using production ones here, so we need to deploy it first.

File: src/components/logo/Logo.jsx [hunk 2]
```diff
@@ -7,9 +7,17 @@ export const BASE_URL = 'https://styleguide.brainly.com/images/logos/';
 
 export type LogoTypeType =
   | 'brainly'
+  | 'brainly-mobile'
```
Comment (matzimowski) [inline src/components/logo/Logo.jsx]: current ones were not exported too.

**Comment by anna-wro** [comment]
please add them to https://styleguide.brainly.com/151.0.5/docs/basics.html#logo as well :)

**Comment by anna-wro** [comment]
@matzimowski please add them to https://styleguide.brainly.com/151.0.5/docs/basics.html#logo as well :)

**Pull request merged**